packages/paste-theme/stories/themeProvider.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {Helmet} from 'react-helmet';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Input} from '@twilio-paste/input';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Select, Option} from '@twilio-paste/select';
+import {Stack} from '@twilio-paste/stack';
+import {TextArea} from '@twilio-paste/textarea';
+
+import {ThemeProvider} from '../src/themeProvider';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Theme/ThemeProvider',
+  component: ThemeProvider,
+};
+
+export const EnglishFontFamily = (): React.ReactNode => (
+  <Box>
+    <Paragraph>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
+      magna aliqua. Nisi porta lorem mollis aliquam ut porttitor leo. Hendrerit gravida rutrum quisque non. A arcu
+      cursus vitae congue mauris rhoncus aenean vel elit. Tortor dignissim convallis aenean et tortor at risus.
+      Vestibulum lorem sed risus ultricies. Tempor nec feugiat nisl pretium fusce id. Morbi tempus iaculis urna id
+      volutpat lacus laoreet non curabitur. In ante metus dictum at. Sit amet risus nullam eget felis eget nunc
+      lobortis.
+    </Paragraph>
+    <Stack orientation="vertical" spacing="space50">
+      <Button variant="primary" onClick={() => {}}>
+        Click me
+      </Button>
+      <Input aria-label="Search" placeholder="Search options..." type="text" />
+      <Select aria-label="Options">
+        <Option value="option1">Option 1</Option>
+      </Select>
+      <TextArea aria-label="Feedback" value="Lorem ipsum dolor sit amet, consectetur adipiscing elit" />
+    </Stack>
+  </Box>
+);
+
+export const JapaneseFontFamily = (): React.ReactNode => (
+  <>
+    <Helmet>
+      <html lang="ja" />
+    </Helmet>
```

Comment:
suggestion: We do have this Storybook plugin installed https://storybook.js.org/addons/storybook-addon-i18n. If it supports it, it might be a better solution to using Helmet.